### DISCUSSION TITLE
Disable Green Download Button

### DISCUSSION THREAD

**Discussion by smatlanta** [aclap-dev/video-downloadhelper#349] [Help and Q&A]
### Your question

How do I disable the popup Green one-click download button using Firefox VDH 

### Addon & CoApp version and details

version: 8.2.2.8a1
channel: beta
build date: 2024-03-14
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: accepted
key (only 16 first characters): 71bccf853bc84fe1
platform: aarch64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:124.0) Gecko/20100101 Firefox/124.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.17",
  "binary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "mac",
    "arch": "arm64",
    "node": "18"
  },
  "home": "/Users/stevemoore",
  "converterBinary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "n6.1-9-g63b936e978"
}

**Comment by Mattmjs** [comment]
What are you referring to exactly ?

**Comment by smatlanta** [comment]
On Mar 31, 2024, at 10:50 PM, Mattmjs ***@***.***> wrote:﻿
What are you referring to exactly ?
